client/pages/Geo/Geo.tsx

```diff
@@ -1,8 +1,36 @@
 import './geo.scss'
-import React from 'react'
+import React, { Suspense, useEffect, useState } from 'react'
+
+import Loading from '@client/components/Loading'
+
+const GeoMap = React.lazy(() => import('./GeoMap'))
 
 const Geo: React.FC = () => {
-  return <div />
+  const [isRenderAllowed, setIsRenderAllowed] = useState(false)
+
+  // makes the map fill remaining screen space nicely
+  useEffect(() => {
+    const main = document.getElementById('main')
+    const mainDefaultVal = main.style.minHeight
+    main.style.minHeight = '100vh'
```

Comment:
Would adding this to main css selector break something? If not , I would add it there , not in JavaScript